.github/workflows/janitor-clusters.yaml

```diff
@@ -26,12 +26,11 @@ jobs:
             workload_identity_provider: "projects/567187841907/locations/global/workloadIdentityPools/staging-shared-9bd2/providers/staging-shared-gha"
 
     steps:
-      - uses: google-github-actions/auth@ef5d53e30bbcd8d0836f4288f5e50ff3e086997d # v1.0.0
```

Comment:
My theory is the v1 one didn't write the file to where the gcloud action looked for it. These versions match the one used in the workflow that creates the cluster.